js/index.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable import/extensions */
+
+import { keysDataEn, keysDataRu } from './data.js';
+import Keyboard from './modules/Keyboard.js';
+import * as helper from './modules/helper.js';
+
+window.onload = () => {
+  const keyboard = new Keyboard();
+
+  keyboard.createKeyboard();
+
+  const insertCurrentLanguage = (data) => {
+    const { lang } = data;
+    if (lang === 'en') {
+      keyboard.paintKeyboard(keysDataEn);
+      helper.pushDataInLocalStorage(keyboard.lang = 'en');
+    }
+    if (lang === 'ru') {
+      keyboard.paintKeyboard(keysDataRu);
+      helper.pushDataInLocalStorage(keyboard.lang = 'ru');
+    }
+  };
+
+  const value = helper.loadDataFromLocalStorage();
+  if (value) {
+    insertCurrentLanguage(value);
+  } else {
+    keyboard.paintKeyboard(keysDataEn);
+    helper.pushDataInLocalStorage(keyboard.lang = 'en');
+  }
+
+  document.addEventListener('click', (event) => {
+    const display = document.getElementById('keyBoardDisplay');
+    let flag = true;
+    if (event.target.type === 'submit') {
+      if (event.target.name === '8') {
```

Comment:
`'8'` look like a magic number. It's better to create some object like `KeyCodes.Backspace` which will return '8'. It will make code more readable and understandable.